code/nathan/python/rot_cipher.py

```diff
@@ -0,0 +1,16 @@
+import string
+
+letters = list(string.ascii_lowercase)
+
+plaintext = input("What would you like to encrypt? ")
+crypto_key = int(input("What amount of rotation would you like to use? "))
+
+encrypted_text = ''
```

Comment:
Really close. Run your program with 13 to test. Check out the error. might give you a hint. `encrypted_text = ??????`

Reply:
Ignore this, I think I made a mistake. Apologies.